src/app/client/src/app/modules/workspace/components/create-training/create-training.component.html

```diff
@@ -0,0 +1,77 @@
+<div class="ui grid mt-5">
```

Comment:
changes required for telemetry interact events pageId

Reply:
done

Reply:
This is not done  still page id doesn't have training prefix